mvpa2/testing/tools.py

```diff
@@ -61,23 +62,28 @@ class SkipTest(Exception):
     assert_string_equal)
 
 
+
```

Comment:
even more spaces??? what for?? two empty lines is imho very sufficient -- think about fellas debugging in a 80x25 terminal remotely ;)